relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -443,13 +443,13 @@ mod tests {
     scrub_sql_test!(
         collapse_partial_column_lists,
         r#"SELECT myfield1, "a"."b", count(*) AS c, another_field, another_field2 FROM table1 WHERE %s"#,
-        "SELECT .., count(*) AS c, .. FROM table1 WHERE %s"
+        "SELECT .., count(*), .. FROM table1 WHERE %s"
     );
 
     scrub_sql_test!(
         collapse_partial_column_lists_2,
         r#"SELECT DISTINCT a, b,c ,d , e, f, g, h, COALESCE(foo, %s) AS "id" FROM x"#,
-        "SELECT DISTINCT .., COALESCE(foo, %s) AS id FROM x"
+        "SELECT DISTINCT .., COALESCE(foo, %s) FROM x"
```

Comment:
@gggritso this will again change a lot of group hashes and thereby mess with the "time spent" column, are we OK with that?

Reply:
I asked in the team channel just in case, but IMO this is OK and we should do it sooner rather than later!